ashionlogin.html

```diff
@@ -15,63 +15,68 @@
-            <span class="navbar-toggler-icon"></span>
-          </button>
-          <div class="collapse navbar-collapse" id="navbarNav">
-            <ul class="navbar-nav">
-              <li class="nav-item">
-                <a class="nav-link active" aria-current="page" href="/index.html">HOME</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="#">WOMEN'S</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="#">MEN'S</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="#">BLOG</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="#">SHOP</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="#">CONTACT</a>
-              </li>
-            </ul>
-          </div>
-          <div>
-            <ul class="navbar-nav">
-                <li>
-                    <div>
-                        <form class="example" action="action_page.php">
-                            <input type="text" placeholder="Search on Fashion.." name="search">
-                            <button type="submit"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-search" viewBox="0 0 16 16">
-                              <path d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"/>
-                            </svg></button>
-                          </form>
-                    </div>
-                </li>
-                <li class="cart">
-                    <a href=""><i class="fa fa-regular fa-heart"></i></i></a>
-                </li>
-                <li class="cart">
-                    <a href=""><i class="fa fa-solid fa-cart-plus"></i></a>
-                </li>
-              </ul>
-              </div>
-            </div>
-          </nav>
-        </div>
-    </nav>
+  <nav class="navbar navbar-expand-lg bg-body-tertiary" id="navbar">
+    <div class="container-fluid">
+      <a href="/index.html">
+        <img src="https://preview.colorlib.com/theme/ashion/img/logo.png">
+      </a>
+      <a class="navbar-brand" href="#"></a>
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarNav">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <a class="nav-link active" aria-current="page" href="/index.html">HOME</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">WOMEN'S</a>
```

Comment:
replace all href="#" to href="javascript:void(0)" or with actual file path

Reply:
resolved